README.md

```diff
@@ -244,6 +245,8 @@ options:
   -d, --decode          Decode advertising data
   -o OUTPUT, --output OUTPUT
                         PCAP output file name
+  -z, --zmq             Enable ZMQ server
+  --zmqsetting          Set ZMQ server ip and port (default:127.0.0.1:4222)
```

Comment:
Adding a space after `default:` should make this more readable (and as it would be rendered in fact?): ... `and port (default: 127.0.0.1:4222)`

Reply:
Changed that accordingly.